### ISSUE TITLE
Support Flutter Web

### ISSUE THREAD

**Issue by josancamon19** [HomeXLabs/pusher-websocket-flutter#21]
The following code is supposed to works for the initialization of pusher **Works in Android and iOS**
```
 await Pusher.init( "KEY", PusherOptions(cluster: "-",), enableLogging: true);
```
With flutter web this initialization code is even unable to initiate, no errors apparently, just the line is never completed and the code gets stopped.

**Comment by MisterJimson** [comment]
This plugin does not support Flutter web, its a wrapper around the iOS and Android SDKs

**Comment by josancamon19** [comment]
Yep, that's what I found in the docs, the thing is that in the pub page says web supported, so that confused me a lot. I'll be waiting for the flutter support.
![Screen Shot 2020-06-01 at 4 04 39 PM](https://user-images.githubusercontent.com/18078879/83454371-b5407d80-a421-11ea-80af-ee8194c5b3d5.png)
